### ISSUE TITLE
Add all accessibility features

### ISSUE THREAD

**Issue by tahsin4466** [Prof-Rosario-UCLA/team19#18]
Must comply with basic accessibility principles (e.g., color contrast, tab navigation, semantic HTML). ARIA attributes should be used where applicable.

_" All attributes will contain ARIA labels, alt text, and adherence to accessibility principles when needed"_
